packages/viewer/tests/cypress/tests-e2e/search/search-results.cy.ts

```diff
@@ -19,11 +20,11 @@ const viewportWidth = Cypress.config('viewportWidth')
 /**
  * Configurable `.then` callback to check element index among siblings.
  *
- * @param {Number} expected The expected index of the tested element.
- * @param {String} [message] The error message in case of failure.
+ * @param {number} expected The expected index of the tested element.
+ * @param {string} [message] The error message in case of failure.
```

Comment:
you can remove the typing of param in JSDoc when you migrate to TS, as you are already typing it directly into the code